### PR TITLE
kfctlClient is a simple utility for sending requests to click to deploy

### PR DESCRIPTION
**Pull request by jlewi**
* This is a simple utility to help debug click to deploy backend.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/3830)

### CODE REVIEW

File: bootstrap/cmd/kfctlClient/main.go [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+	"github.com/pkg/errors"
+	"golang.org/x/oauth2"
+	"os"
+
+	// log "github.com/sirupsen/logrus"
+	"golang.org/x/oauth2/google"
+	dm "google.golang.org/api/deploymentmanager/v2"
+)
+
+func init() {
```
Comment (gabrielwen) [inline bootstrap/cmd/kfctlClient/main.go]: do we still need this?

File: bootstrap/cmd/kfctlClient/main.go [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+// NewServerOption creates a new CMServer with a default config.
+func NewServerOption() *ServerOption {
+	s := ServerOption{}
+	return &s
+}
+
+// AddFlags adds flags for a specific Server to the specified FlagSet
+func (s *ServerOption) AddFlags(fs *flag.FlagSet) {
+
+	fs.StringVar(&s.Config, "config", "", "Path to a YAML file describing an app to create on startup.")
```
Comment (gabrielwen) [inline bootstrap/cmd/kfctlClient/main.go]: is there a config you will point to?  I might be able to use it for load test as well, or at least for my reference.

Comment (kunmingg) [reply bootstrap/cmd/kfctlClient/main.go]: I think we need to use a config file available from release tag.

File: bootstrap/cmd/kfctlClient/main.go (hunk 1 above)
Comment (jlewi) [reply bootstrap/cmd/kfctlClient/main.go]: No removed it.

File: bootstrap/cmd/kfctlClient/main.go (hunk 2 above)
Comment (jlewi) [reply bootstrap/cmd/kfctlClient/main.go]: You can specify the URI of what ever KfDef file you want to use.
I set a default value to point to the GCP value on master.

**Comment by jlewi** [comment]
@gabrielwen @kunmingg Sorry for the delay. This is ready for another look.

**Pull request merged**